db/migrations/000012_add_gps_coords.down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE infos DROP COLUMN "longitude" text;
+ALTER TABLE infos DROP COLUMN "latitude" text;
+ALTER TABLE infos DROP COLUMN "location" text;
```

Comment:
`rgeo` seems to be fast enough that the location text itself might not even need to be cached? In my quick profiling the reverse geocoding almost doesn't show up. Instead it could happen at "layout-time".